docs/05_operations/02_commands/04_genx-commands.md

```diff
@@ -0,0 +1,398 @@
+---
+title: 'Operations - Genx commands'
+sidebar_label: 'Genx commands'
+id: genx-commands
+keywords: [operations, genx, commands]
+tags:
+    - genx
+    - cli
+    - commands
+    - terminal
+---
+<!-- Genx  -->
+
+Genesis provides a `genx` which is a CLI scaffolding tool that enables you to create Genesis applications from scratch. This interface allows you to create a new project based on some existing seeds, so you do not need to worry about creating a new project from scratch.
+
```

Comment:
I'd consider wording it along these lines:

Genx CLI is a tool to simplify and speed up local development. It provides the following functionality:
* enables you to scaffold Genesis applications from scratch based on existing seeds
* start local development server
* build artifacts for production
* execute unit and end-to-end tests and collect coverage data
* perform code linting
* etc.

Reply:
duly considered